torch/csrc/autograd/compiled_autograd.cpp

```diff
@@ -0,0 +1,25 @@
+#include <torch/csrc/autograd/compiled_autograd.h>
+#include <torch/csrc/autograd/generated/Functions.h>
+
+// see [Note: Compiled Autograd]
+
+namespace torch {
+namespace autograd {
+
+void CompiledNodeArgs::collect(torch::autograd::generated::TypeAndSize& t) {
```

Comment:
Why have these in cpp? You don't want them inlined?

Reply:
Ho? Could you clarify what you mean by that?

Reply:
`TypeAndSize` is defined in the generated files, which import `compiled_autograd.h` for the definitions of the visitors.